io.github.YouROK.TorrServer.yml

```diff
@@ -0,0 +1,39 @@
+app-id: io.github.YouROK.TorrServer
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+finish-args:
+  - --share=network
+modules:
+  - name: TorrServer
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 TorrServer /app/bin/TorrServer
+      - install -Dm644 io.github.YouROK.TorrServer.appdata.xml /app/share/metainfo/io.github.YouROK.TorrServer.metainfo.xml
+      - install -Dm644 io.github.YouROK.TorrServer.png /app/share/icons/hicolor/512x512/apps/io.github.YouROK.TorrServer.png
+      - install -Dm644 io.github.YouROK.TorrServer.desktop /app/share/applications/io.github.YouROK.TorrServer.desktop
+      - install -Dm755 run.sh /app/bin/run.sh
+
+    sources:
+      - type: file
+        url: https://github.com/YouROK/TorrServer/releases/download/MatriX.134/TorrServer-linux-amd64
+        dest-filename: TorrServer
+        sha256: 583ca01bd6869c3db49c6c6d1b4b7e4f6284c1b27e7d9d2e45a0a7762aebb172
```

Comment:
linux arm64 is available https://github.com/YouROK/TorrServer/releases/download/MatriX.134/TorrServer-linux-arm64